src/flashbar/interfaces.ts

```diff
@@ -40,6 +40,69 @@ export namespace FlashbarProps {
     warningIconAriaLabel?: string;
   }
 
+  export interface Style {
+    item?: {
+      root?: {
```

Comment:
this also needs `focusRing`
![Screenshot 2025-06-12 at 11 53 44](https://github.com/user-attachments/assets/aef8b3b9-af20-4b90-89b4-0ca2d093b486)
, so that a focused flash item can have an accessible focus ring color set

Reply:
Ah! I didn't know that was focusable. Adding...